geonode_mapstore_client/client/js/epics/gnsave.js

```diff
@@ -138,14 +142,42 @@ export const gnSaveContent = (action$, store) =>
                             'title': action.metadata.name,
                             'abstract': action.metadata.description,
                             'thumbnail_url': action.metadata.thumbnail
-                        })
+                        }),
+                        action.showNotifications && successNotification({title: "saveDialog.saveSuccessTitle", message: "saveDialog.saveSuccessMessage"})
                     );
                 })
                 .catch((error) => {
-                    return Observable.of(saveError(error.data || error.message));
+                    return Observable.of(
+                        saveError(error.data || error.message),
+                        action.showNotifications && errorNotification({title: "errorTitleDefault", message: "errorDefault"})
```

Comment:
Are `errorTitleDefault` and `errorDefault` valid message id from MapStore or  they are new?

For the message we could use `map.mapError.errorDefault` https://github.com/GeoNode/geonode-mapstore-client/blob/master/geonode_mapstore_client/client/js/plugins/save/SaveModal.jsx#L100